src/applications/mhv/medications/util/constants.js

```diff
@@ -68,7 +68,6 @@ export const rxListSortingOptions = [
 ];
 
 export const medicationsUrls = {
-  prescriptionsUrl: '/my-health/medications/',
-  medicationsLogin:
-    '/my-health/medications/prescriptions?next=loginModal&oauth=true',
+  prescriptionsUrl: '/my-health/medications',
```

Comment:
what are your thoughts on keeping the naming convention the same, instead of using prescriptions vs medications, maybe they can both be medications like this
`medicationsUrl: ..`
`medicationsLogin: ..`